infra/relayer/src/purse.js

```diff
@@ -59,6 +59,7 @@ const REDIS_PENDING_PREFIX = `pending_tx_`
 const REDIS_PENDING_TX_PREFIX = `pending_txobj_`
 const JSONRPC_QPS = 100
 const JSONRPC_MAX_CONCURRENT = 25
+const ACCOUNT_ACQUISITION_TIMEOUT = 15000
```

Comment:
Super nit for when someone looks is on-call and gets woken up because nothing is working: add a comment mentioning the unit is msec or change naming to ACCOUNT_ACQUISITION_TIMEOUT_MSEC

Reply:
Considering all time-based units in JS are in millisecond, is this really something we need to comment every time?  I wonder if it shouldn't be the other way around, comment if it's not millisecons.

Reply:
I personally favor being explicit for all units.
My preference comes from years ago working in the space industry... See https://www.simscale.com/blog/2017/12/nasa-mars-climate-orbiter-metric/ :)

Reply:
Hungarian notation bothers me even more so I'll add a comment under protest. :black_flag: